certif/app/models/member.js

```diff
@@ -1,10 +1,23 @@
 import Model, { attr } from '@ember-data/model';
 import { memberAction } from 'ember-api-actions';
+import { service } from '@ember/service';
 
 export default class Member extends Model {
+  @service intl;
+
   @attr('string') firstName;
   @attr('string') lastName;
   @attr('boolean') isReferer;
+  @attr('string') role;
+
+  certificationCenterMembersRole = {
```

Comment:
```suggestion
  #certificationCenterMembersRole = {
```
On pourrait utiliser un attribut privé. Ce n'est pas nécessaire mais c'est simplement une suggestion 😄